dune-project

```diff
@@ -0,0 +1,61 @@
+(lang dune 2.7)
+(generate_opam_files true)
+(allow_approximate_merlin)
+
+(name repr)
+(source (github mirage/repr))
+(license ISC)
+(authors "Thomas Gazagnaire")
+(maintainers "thomas@gazagnaire.org")
+
+(package
+ (name repr)
+ (depends
+  (ocaml (>= 4.08.0))
+  (fmt (>= 0.8.0))
+  uutf
+  (jsonm (>= 1.0.0))
+  (base64 (>= 2.0.0))
+  (hex :with-test)
+  (alcotest (and (>= 1.1.0) :with-test)))
+ (synopsis "Dynamic type representaitons. Provides no stability guarantee")
+ (description "\
+!!!!!!!!!!!!!!!!!!!!
```

Comment:
This is a little bit excitable. Perhaps tone it down with a notice that compatibility is not yet guaranteed (the last sentence is fine) instead of the exclamation points.

Reply:
Ahah, yes. It was copied from `dune-internal-libs`.